File: services/problems/1/boilerplate_full/function.py
from typing import List

**Your Code Goes Here**

if __name__ == "__main__":
    n_arr = int(input())
arr = list(map(int, input().split()))
target = int(input())
    
    result = twoSum(arr, target)
    
    print(' '.join(map(str, result)))

File: services/problems/1/boilerplate_full/function.js
**Your Code Goes Here**

const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

const lines = [];
rl.on('line', (line) => {
    lines.push(line);
}).on('close', () => {
    const n_arr = parseInt(lines[0]);
    const arr = lines[1].split(' ').map(Number);
    const target = parseInt(lines[2]);
    
    const result = twoSum(arr, target);
    
    console.log(result.join(' '));
});